test/python/transpiler/test_sabre_swap.py

```diff
@@ -1329,9 +1329,9 @@ def setUpClass(cls):
         super().setUpClass()
         cls.backend = Fake27QPulseV1()
         cls.backend.configuration().coupling_map = MUMBAI_CMAP
+        cls.backend.configuration().basis_gates += ["for_loop", "while_loop", "if_else"]
```

Comment:
I realized that this test wasn't actually updating the backend basis gates, and this made the new target-based path fail with a "control flow gate no found" error. One of the oversights from the old model.

Reply:
Good catch!